src/commands/dev/dev.js

```diff
@@ -200,17 +207,17 @@ const startFrameworkServer = async function ({ settings }) {
 
   runCommand(settings.command, settings.env, spinner)
 
+  let port
   try {
-    const open = await waitPort({
+    port = await waitPort({
       port: settings.frameworkPort,
-      // Cannot use `localhost` as it may point to IPv4 or IPv6 depending on node version and OS
-      host: '127.0.0.1',
+      host: 'localhost',
```

Comment:
@danez @eduardoboucas this has caused a regressions for one of our own CI pipelines https://netlify.slack.com/archives/CPJQSPQE4/p1665509398672569

Could we use `settings.frameworkHost` here to solve this?